zaas-service/src/main/java/org/zowe/apiml/zaas/security/config/NewSecurityConfiguration.java

```diff
@@ -612,12 +625,11 @@ public WebSecurityCustomizer webSecurityCustomizer() {
         @Bean
         public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
             return baseConfigure(http.securityMatchers(matchers -> matchers.requestMatchers("/**", "/gateway/version")))
-                    .authorizeHttpRequests(requests -> requests
-                            .anyRequest()
-                            .permitAll()).logout(logout -> logout.disable())
-                    // sort out client and apiml internal certificates
-                    .addFilterBefore(new CategorizeCertsFilter(publicKeyCertificatesBase64, certificateValidator), AnonymousAuthenticationFilter.class)
-                    .build();
+                .authorizeHttpRequests(requests -> requests
+                    .anyRequest()
+                    .permitAll()).logout(AbstractHttpConfigurer::disable)
+                // sort out client and apiml internal certificates
```

Comment:
I think this comment is no longer valid if the categorizeCertsFilter is not applied